config/instana/templates/instana.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: instana.crossplane.io/v1alpha1
+kind: Instana
+metadata:
+  name: instana
+spec:
+  forProvider:
+    instanaVersion: {{.Values.spec.instanaVersion}}
+    nfsServerHost: {{.Values.spec.nfsServerHost}}
+    settingsConfigmap:
+      name: {{.Values.spec.settingsConfigmapName}}
+      namespace: {{.Values.spec.settingsConfigmapNS}}
+  providerConfigRef:
+    name: {{.Values.spec.providerConfigRef}}
```

Comment:
remove this 
![image](https://user-images.githubusercontent.com/4461983/139644509-c80d9252-d546-4612-ab6a-bd58cc943ee6.png)


Ditto for others

Reply:
Do not want to block you, you can create another PR to address

Reply:
I was wondering if user has multiple env , it may need multiple providerconfig to address different cluster